Source/WTF/wtf/generic/RunLoopGeneric.cpp

```diff
@@ -275,7 +280,7 @@ void RunLoop::wakeUp()
 
 RunLoop::CycleResult RunLoop::cycle(RunLoopMode)
 {
-    RunLoop::current().runImpl(RunMode::Iterate);
+    iterate();
```

Comment:
As you can see here, RunLoop::cycle means:
"Run one iteration of runloop, do not wait for new messages."
This is consistent on all other platforms than windows:
Cocoa, GLib, Generic (if I understand the code correctly -- I'm not 100% sure).